TestExitCodeWithSeveralApps fails occasionally

We've seen some occasional failures of this test on Jenkins:

```
16:28:20 === RUN   TestExitCodeWithSeveralApps
16:28:31 --- FAIL: TestExitCodeWithSeveralApps (11.35s)
16:28:31 	rkt_tests.go:115: Running command: /home/fedora/workspace/rkt-matrix/dummy/dummy1/label/fedora-22-flavor-coreos/builds/build-rkt-coreos/build-rkt-1.8.0+git/tmp/functional/rkt --dir=/tmp/datadir-230509682 --local-config=/tmp/localdir-073447721 --system-config=/tmp/systemdir-172121716 --user-config=/tmp/userdir-198816067 --debug --insecure-options=image run --mds-register=false /home/fedora/workspace/rkt-matrix/dummy/dummy1/label/fedora-22-flavor-coreos/builds/build-rkt-coreos/build-rkt-1.8.0+git/tmp/functional/test-tmp/rkt-inspect-exit-0.aci /home/fedora/workspace/rkt-matrix/dummy/dummy1/label/fedora-22-flavor-coreos/builds/build-rkt-coreos/build-rkt-1.8.0+git/tmp/functional/test-tmp/rkt-inspect-exit-1.aci /home/fedora/workspace/rkt-matrix/dummy/dummy1/label/fedora-22-flavor-coreos/builds/build-rkt-coreos/build-rkt-1.8.0+git/tmp/functional/test-tmp/rkt-inspect-exit-2.aci
16:28:31 	rkt_exit_test.go:71: Could not start the app (#2): image: using image from file /home/fedora/workspace/rkt-matrix/dummy/dummy1/label/fedora-22-flavor-coreos/builds/build-rkt-coreos/build-rkt-1.8.0+git/tmp/functional/stage1-coreos.aci
16:28:31 		image: using image from file /home/fedora/workspace/rkt-matrix/dummy/dummy1/label/fedora-22-flavor-coreos/builds/build-rkt-coreos/build-rkt-1.8.0+git/tmp/functional/test-tmp/rkt-inspect-exit-0.aci
16:28:31 		image: using image from file /home/fedora/workspace/rkt-matrix/dummy/dummy1/label/fedora-22-flavor-coreos/builds/build-rkt-coreos/build-rkt-1.8.0+git/tmp/functional/test-tmp/rkt-inspect-exit-1.aci
16:28:31 		image: using image from file /home/fedora/workspace/rkt-matrix/dummy/dummy1/label/fedora-22-flavor-coreos/builds/build-rkt-coreos/build-rkt-1.8.0+git/tmp/functional/test-tmp/rkt-inspect-exit-2.aci
16:28:31 		stage0: Preparing stage1
16:28:31 		stage0: Writing image manifest
16:28:31 		stage0: Loading image sha512-4d91efc2dfd948b76fe0ebc4632838e10fc23720173543289ed0685f42ac7ca8
16:28:31 		stage0: Writing image manifest
16:28:31 		stage0: Loading image sha512-9082ad338defb6b1b051071fe6498661c8dd3f76446ff2d1bca1f3441d84534a
16:28:31 		stage0: Writing image manifest
16:28:31 		stage0: Loading image sha512-dd419198d9e0b291403e2d13ffaddfe510d8d081556dd995a3fcf3e18ab5a160
16:28:31 		stage0: Writing image manifest
16:28:31 		stage0: Writing pod manifest
16:28:31 		stage0: Setting up stage1
16:28:31 		stage0: Wrote filesystem to /tmp/datadir-230509682/pods/run/b0d89d53-a33a-4549-95b6-f663eb315226
16:28:31 		stage0: Pivoting to filesystem /tmp/datadir-230509682/pods/run/b0d89d53-a33a-4549-95b6-f663eb315226
16:28:31 		stage0: Execing /init
16:28:31 		networking: loading networks from /tmp/localdir-073447721/net.d
16:28:31 		networking: loading network default with type ptp
16:28:31 		Spawning container rkt-b0d89d53-a33a-4549-95b6-f663eb315226 on /tmp/datadir-230509682/pods/run/b0d89d53-a33a-4549-95b6-f663eb315226/stage1/rootfs.
16:28:31 		Press ^] three times within 1s to kill container.
16:28:31 		systemd 229 running in system mode. (+PAM +AUDIT +SELINUX +IMA -APPARMOR +SMACK -SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT -GNUTLS -ACL +XZ -LZ4 +SECCOMP +BLKID -ELFUTILS +KMOD -IDN)
16:28:31 		Detected virtualization rkt.
16:28:31 		Detected architecture x86-64.
16:28:31 		
16:28:31 		Welcome to [1mLinux[0m!
16:28:31 		
16:28:31 		Set hostname to <rkt-b0d89d53-a33a-4549-95b6-f663eb315226>.
16:28:31 		Initializing machine ID from container UUID.
16:28:31 		[[0;32m  OK  [0m] Created slice system.slice.
16:28:31 		[[0;32m  OK  [0m] Created slice system-prepare\x2dapp.slice.
16:28:31 		[[0;32m  OK  [0m] Started Pod shutdown.
16:28:31 		[[0;32m  OK  [0m] Started hello2 Reaper.
16:28:31 		[[0;32m  OK  [0m] Started hello0 Reaper.
16:28:31 		[[0;32m  OK  [0m] Listening on Journal Socket (/dev/log).
16:28:31 		[[0;32m  OK  [0m] Listening on Journal Socket.
16:28:31 		         Starting Journal Service...
16:28:31 		[[0;32m  OK  [0m] Started hello1 Reaper.
16:28:31 		         Starting Create /etc/passwd and /etc/group...
16:28:31 		[[0;32m  OK  [0m] Started Create /etc/passwd and /etc/group.
16:28:31 		[[0;32m  OK  [0m] Started Journal Service.
16:28:31 		         Starting Prepare minimum environment for chrooted applications...
16:28:31 		         Starting Prepare minimum environment for chrooted applications...
16:28:31 		         Starting Prepare minimum environment for chrooted applications...
16:28:31 		[[0;32m  OK  [0m] Started Prepare minimum environment for chrooted applications.
16:28:31 		[[0;32m  OK  [0m] Started Application=hello1 Image=hello1.
16:28:31 		[[0;32m  OK  [0m] Started Prepare minimum environment for chrooted applications.
16:28:31 		[[0;32m  OK  [0m] Started Prepare minimum environment for chrooted applications.
16:28:31 		[[0;32m  OK  [0m] Started Application=hello0 Image=hello0.
16:28:31 		[[0;32m  OK  [0m] Started Application=hello2 Image=hello2.
16:28:31 		[[0;32m  OK  [0m] Reached target rkt apps target.
16:28:31 		[ 1099.566857] hello1[7]: HelloWorld
16:28:31 		
16:28:31 		[ 1099.606931] hello2[9]: HelloWorld
16:28:31 		
16:28:31 		Sending SIGTERM to remaining processes...
16:28:31 		Sending SIGKILL to remaining processes...
16:28:31 		Container rkt-b0d89d53-a33a-4549-95b6-f663eb315226 failed with error code 5.
```

It seems the pod exits before `hello0` has a chance to print "HelloWorld", that is, `hello1` exits before `hello0` prints anything, causing the pod to exit and the test to fail.

We should probably sleep in every app, and sleep a bit more in `hello2`.

--- a/tests/rkt_exit_test.go
+++ b/tests/rkt_exit_test.go
@@ -46,15 +46,15 @@ func TestExitCodeSimple(t *testing.T) {
 // exit codes.
 func TestExitCodeWithSeveralApps(t *testing.T) {
 	image0File := patchTestACI("rkt-inspect-exit-0.aci", "--name=hello0",
-		"--exec=/inspect --print-msg=HelloWorld --exit-code=0")
+		"--exec=/inspect --print-msg=HelloWorld --exit-code=0 --sleep=1")
 	defer os.Remove(image0File)
 
 	image1File := patchTestACI("rkt-inspect-exit-1.aci", "--name=hello1",
-		"--exec=/inspect --print-msg=HelloWorld --exit-code=5")
+		"--exec=/inspect --print-msg=HelloWorld --exit-code=5 --sleep=1")
 	defer os.Remove(image1File)
 
 	image2File := patchTestACI("rkt-inspect-exit-2.aci", "--name=hello2",
-		"--exec=/inspect --print-msg=HelloWorld --exit-code=6 --sleep=1")
+		"--exec=/inspect --print-msg=HelloWorld --exit-code=6 --sleep=2")
 	defer os.Remove(image2File)
 
 	ctx := testutils.NewRktRunCtx()

